src/SpaceVecAlg/AdmittanceVec.h

```diff
@@ -0,0 +1,181 @@
+// Copyright 2012-2018 CNRS-UM LIRMM, CNRS-AIST JRL
+//
+// This file is part of SpaceVecAlg.
+//
+// SpaceVecAlg is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// SpaceVecAlg is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have receaved a copy of the GNU Lesser General Public License
+// along with SpaceVecAlg.  If not, see <http://www.gnu.org/licenses/>.
+
+#pragma once
+
+namespace sva
+{
+
+using namespace Eigen;
+
+/**
+	* Admittance Vector.
+	* Mechanical admittance is a measure of how much a structure moves
+	* in response to a given force:
+	*
+	*     v = A * F
+	*
+	* where F is the exerted force, A the admittance and v the resulting output
+	* velocity. Admittance is the reciprocal of mechanical impedance; see
+	* <https://en.wikipedia.org/wiki/Mechanical_impedance>.
+	*/
+template<typename T>
+class AdmittanceVec
+{
+public:
+	typedef Vector3<T> vector3_t;
+	typedef Vector6<T> vector6_t;
+
+	friend class PTransform<T>;
+
+public:
+	/// Zero admittance vector
+	static AdmittanceVec<T> Zero()
+	{
+		return AdmittanceVec<T>(vector3_t::Zero(), vector3_t::Zero());
+	}
+
+public:
+	AdmittanceVec():
```

Comment:
Missing doxygen

Reply:
Done in d18080deb6bc9107ad2bc95bb925f61f9948bc4d